libmogwai-schedule/schedule-service.c

```diff
@@ -510,6 +510,26 @@ entries_changed_cb (MwsScheduler *scheduler,
                         (GCallback) entry_notify_cb, self);
     }
 
+  /* Emit Removed signals for entries. */
+  for (gsize i = 0; removed != NULL && i < removed->len; i++)
+    {
+      MwsScheduleEntry *entry = MWS_SCHEDULE_ENTRY (removed->pdata[i]);
+      g_autoptr(GError) local_error = NULL;
+
```

Comment:
I'd say there's no need for this newline but I don't mind if you did this on purpose.

Reply:
It was on purpose.